tests/testthat/test-RInterface.R

```diff
@@ -6,6 +6,55 @@
 library(BinSeg)
 set.seed(300)
 
+psd <- function(x) {
+  sqrt(sum((x - mean(x))^2) / (length(x)))
+}
+
+fixed_mean_psd <- function(x, mean){
+  sqrt(sum((x - mean)^2) / (length(x)))
+}
+
+check_resid <- function(object, ncpts=nrow(object@models_summary)){
+
+  dist <- dist(object)
+  obj_coef <- coef(object, ncpts)
+  resids <- resid(object)
+  data <- object@data
+
+  if (dist == "mean_norm"){ # Cost function per segment.
```

Comment:
here you can simplify the code by changing
```r
if(dist=="mean_norm")resid_func <- something
else if(dist=="var_norm")resid_func <- something_else
```
you can instead do
```r
resid_func <- if(dist=="mean_norm")something else if(dist=="var_norm") something_else
```

Reply:
Done! Thanks tdhock :)